addon/mixins/oidc-application-route-mixin.js

```diff
@@ -2,6 +2,22 @@ import ApplicationRouteMixin from "ember-simple-auth/mixins/application-route-mi
 import Mixin from "@ember/object/mixin";
 
 export default Mixin.create(ApplicationRouteMixin, {
+  sessionAuthenticated() {
+    /**
+     * If there is a stored `continueTransition` URL in the localstorage
+     * transition to it otherwise call the parent/super to invoke the normal
+     * behavior of the `sessionAuthenticated` hook.
+     */
+    const continueTransition = this.get("session.data.continueTransition");
+    this.set("session.data.continueTransition", undefined);
+
+    if (continueTransition) {
+      this.transitionTo(continueTransition);
+    } else {
+      this._super();
+    }
```

Comment:
```
    if (continueTransition) {
      return this.transitionTo(continueTransition);
    }

    return this._super();
```

Seems cleaner to me

Reply:
I didn't do it this way because this method normally (as defined in the ember-simple-auth addon) returns nothing. If we do it this the sometimes a transition object is returned. That is why I thought it is not really cleaner. Do you still think we should change this?

Reply:
Fair enough!